packages/swap/src/config/env.ts

```diff
@@ -43,6 +43,26 @@ export default z
     MAINTENANCE_MODE: optionalBoolean,
     LIQUIDITY_WARNING_THRESHOLD: optionalNumber(-5),
     COINGECKO_API_KEY: z.string().optional(),
+    DCA_USD_CHUNK_SIZE: optionalString('{}').transform((string) => {
+      try {
+        return z
+          .array(
+            z.object({
+              asset: z.nativeEnum(InternalAssets),
+              usdChunkSize: z.number(),
+            }),
+          )
+          .parse(JSON.parse(string));
+      } catch (err) {
+        const error = err as Error;
+        // eslint-disable-next-line no-console
+        console.warn({
+          message: `Could not parse DCA_USD_CHUNK_SIZE variable. error: "${error?.message}"`,
+        });
+        return undefined;
+      }
+    }),
+    DCA_CHUNK_INTERVAL_SECONDS: optionalNumber(2),
```

Comment:
think the protocol uses a number of blocks instead of seconds

```suggestion
    DCA_CHUNK_INTERVAL_BLOCKS: optionalNumber(2),
```